proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,68 @@
+SDL must log corresponding error internally continue work as assigned (due to existing requirements).
+10. In case mobile app does NOT send SetGlobalProperties_request at all with <vrHelp> and <helpPrompt> during 10 sec timer and <vrHelp> and <helpPrompt> were NOT resumed within data resumption process (at SetGlobalProperties) and this mobile app has successfully registered AddCommands and/or DeleteCommands requests (resumed within data resumption).
+SDL must provide the value of <helpPrompt> and <vrHelp> from internal list based on registered AddCommands and DeleteCommands requests to HMI.
+11. In case SDL transfers AddCommand and/or DeleteCommand requests from mobile app to HMI and SDL receives any <errorCode> at response from HMI.
+SDL must transfer received <errorCode> from HMI to mobile app.
+SDL must NOT update internal list with "vrHelp" and "helpPrompt" params.
+12. In case mobile app sends SetGlobalProperties_request during 10 sec timer: without <helpPrompt> and/or <vrHelp> params, other valid params related to this RPC and this mobile app has NO registered VR.AddCommands and/or VR.DeleteCommands requests (resumed during data resumption).
+SDL must provide the default values of <helpPrompt> and <vrHelp> to HMI.
+13. In case mobile app sends ResetGlobalProperties_request to SDL.
+SDL must send default values of <vrHelp> and <helpPrompt> param to HMI (via UI/TTS.SetGlobalProperties), continue update internal list with "vrHelp" and "helpPrompt" parameters with new requested AddCommands and/or DeleteCommands till mobile app sends SetGlobalProperties request with valid <vrHelp> and <helpPrompt> params to SDL.
+SDL must NOT send by itself UI/TTS.SetGlobalProperties request with updated <vrHelp> and <helpPrompt> to HMI.
+14. In case SetGlobalProperties with <vrHelp> and <helpPrompt> params was successfully resumed within data resumption process and mobile app either does NOT send SetGlobalProperties_request during 10 sec timer or sends SetGlobalProperties_request without <vrHelp> AND <helpPrompt> params during 10 sec timer.
+SDL must continue creating of internal list with "vrHelp" and "helpPrompt" based on successfully registered AddCommands and/or DeleteCommands requests.
+SDL must NOT send SetGlobalProperties request with <vrHelp> and <helpPrompt> to HMI (with values from internal list OR default values) (meaning: HMI already received <vrHelp> and <helpPrompt> during data resumption and SDL should NOT re-write these values by sending SetGlobalProperties by itself).
+
+
+## Potential downsides
+
+N/A
+
+## Impact on existing code
+
+Application of described solution will impact on SDL behavior ralated to app registration with changing HMI level from NONE, resumption procedure.
```

Comment:
@GetmanetsIrina could be rephrased just as:
Application registration process, data resumption procedure.

Reply:
Changed locally.